_summaries/shi2018pointrcnn_2.md

```diff
@@ -1,19 +1,124 @@
+
+A ground-truth box is matched with a proposal if their IoU is above 0.55. The ground-truth box also goes through the canonical transformation. To do refinement, the proposal's formulation is the same as in stage 1, but with smaller bins for proposal center and orientation. The refinement loss is defined as:
+
+<p align="center">
+    <img src="shi2018pointrcnn/refinement_loss.png" width="20%" height="20%">
+</p>
+
+Where $$\mathcal{B}$$ is the set of 3D proposals from stage-1 and $$\mathcal{B}_{pos}$$ are the positive (>0.55 IoU) proposals for regression. $$\text{prob}_{i}$$ denotes the estimated confidence of the bounding box. Eventually, another NMS is applied to remove overlapping bounding boxes with an IoU threshold of 0.01.
+
 * What interesting variants are explored?
 
+**GT-AUG**: To simulate objects with various environments, the authors copied several ground-truth boxes and their inside points from some scenes to same locations of other scenes. This slightly improved mAP on the validation set.
+
+* How well does the paper perform?
+
+PointRCNN is evaluated on the test and validation 3D object detection benchmark of KITTI dataset ([Geiger et al., 2012](http://www.cvlibs.net/datasets/kitti/)). The method outperforms previous SoTAs on car and cyclist detections, but comparable to SoTAs on pedestrian detections. The authors claims that this is due to other methods having multiple sensors for small objects like pedestrians, but PointRCNN only used point clouds, which had sparse points for pedestrians.
+
+<p align="center">
+    <img src="shi2018pointrcnn/table1.png" width="50%" height="50%">
+</p>
+
+PointRCNN also achieved good results on the validation set (on the left). Note that PointRCNN achieved 96% recall with only 50 proposals, which is quite high (on the right).
+
+<p align="center">
+    <img src="shi2018pointrcnn/val_roi_recall.png" width="50%" height="50%">
+</p>
+
+**Ablations**
+
+Ablations on removing canonical transformation (CT) and semantic features (RPN features):
+
+<p align="center">
+    <img src="shi2018pointrcnn/ablate1.png" width="25%" height="25%">
+</p>
+
+Not enlarging stage-1 proposal boxes decreased performance on hard objects:
+
+<p align="center">
+    <img src="shi2018pointrcnn/ablate2.png" width="20%" height="20%">
+</p>
+
+Full bin-based loss (BB-loss) outperformed residual-based loss and other losses in convergence:
+
+<p align="center">
+    <img src="shi2018pointrcnn/ablate3.png" width="30%" height="30%">
+</p>
+
+
 ## TL;DR
```

Comment:
Great summary! You do an excellent job conveying the technical aspects of the paper. Maybe add some examples of the qualitative results from Figure 6 to help visualize what the model is outputting.